CMakeLists.txt

```diff
@@ -236,7 +236,11 @@ elseif(CMAKE_C_COMPILER_ID MATCHES "GNU" OR CMAKE_C_COMPILER_ID MATCHES "Clang")
         endif()
     endif()
     if(MINGW)
-        list(APPEND WARNFLAGS_DISABLE -Wno-pedantic-ms-format)
+        # Add `-Wno-pedantic-ms-format` only if the toolchain is GCC MinGW-w64
+        # llvm-mingw does not support '-Wno-pedantic-ms-format'
+        if (CMAKE_C_COMPILER_ID MATCHES "GNU")
```

Comment:
Extra space after `if`.

Reply:
I applied your review to [e0dd69ad3d52d4b0f2a2952cbe44df5bd4466c4d](https://github.com/zlib-ng/zlib-ng/pull/1570/commits/9708ad83a50d352813c43cf37e452ce9cbd65f43) (Related commit squashed to one commit).